src/Adapters/EventSource.Net46/EventSourceListeningRequest.cs

```diff
@@ -0,0 +1,72 @@
+﻿// -----------------------------------------------------------------------
```

Comment:
P.S. The standard way I have always used is the one that StyleCop reports with default settings.
`//-----------------------------------------------------------------------
// <copyright file="HomeController.cs" company="Microsoft">
//     Copyright (c) Microsoft Corporation. All rights reserved.
// </copyright>
//-----------------------------------------------------------------------
`